contract-tests/stream_entity.rb

```diff
@@ -0,0 +1,61 @@
+require 'ld-eventsource'
+require 'json'
+require 'net/http'
+
+set :port, 8000
+set :logging, false
+
+streams = {}
+streamCounter = 0
```

Comment:
I don't think these two are needed anymore.

Reply:
Indeed, that's a copy-paste mistake.